migrations/20190830121917-enforce-users-emails.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    // All users without email currently in the DB have no transactions nor expenses, thus
+    // it should be safe to delete them all directly.
+    await queryInterface.sequelize.query(`
+      UPDATE  "Users"
+      SET     "deletedAt" = NOW()
+      WHERE   "email" IS NULL
+      AND     "deletedAt" IS NULL
+    `);
```

Comment:
As discussed yesterday, we should also process the matching `Collective type=user`